configurations/create_imaqal_database.py

```diff
@@ -0,0 +1,190 @@
+from dateutil.parser import isoparse
+from src.pipeline_configuration_spec import *
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="CREATE-IMAQAL-POOL",
+    project_start_date=isoparse("2020-01-01T00:00:00+03:00"),
+    project_end_date=isoparse("2021-12-31T00:00:00+03:00"),
+    description="Creates the initial Imaqal Pool from demographics responses to IMAQAL, IMAQAL_COVID19, "
```

Comment:
How long did imaqal run until? Given the cut-off date do we even need to consider any data from the first imaqal project?

Reply:
Resolved offline.